themes/10up-block-theme/assets/css/utilities/visually-hidden.css

```diff
@@ -0,0 +1,10 @@
+.visually-hidden {
```

Comment:
I would add this to the mixins and then call that here.  It's very handy to have available in your styles vs only as a utility class.

```css
@define-mixin visually-hidden {
        border: 0;
	clip: rect(0, 0, 0, 0);
	height: 1px;
	margin: -1px;
	overflow: hidden;
	padding: 0;
	position: absolute;
	width: 1px;
}

.visually-hidden {
        @mixin visually-hidden;
}
```